rules/windows/process_creation/proc_creation_win_hktl_certify.yml

```diff
@@ -27,11 +28,11 @@ detection:
             - '.exe download '
     selection_cli_options:
         CommandLine|contains:
-            - 'vulnerable '
-            - 'template'
-            - 'altname'
-            - 'domain'
-            - 'path'
+            - ' /vulnerable '
```

Comment:
```suggestion
            - ' /vulnerable'
```

Reply:
Reducing this space to cover an additional case. Hope this doesn't conflict with the FP that was found? Let me know

Reply:
no, it's fine.